blog/_posts/2017-11-10-linux-foundation-OSSummit.md

```diff
@@ -0,0 +1,41 @@
+
+<img src="/img/blog/2017/linus-torvalds-on-stage-ossummit.jpg" alt="Linus Torvalds on stage OSSummit EU"/>
+<div class="image-credits">Dirk Hohndel in conversation with Linus Torvalds (Image Credit: official OSSummit photographer)</div>
+
+On the fourth day, there was an optional Diversity Summit which I happily attended. Unfortunately, I don't have a photo from that event, so here is a photo of all the attendees at the Women in Open Source lunch!
+
+<img src="/img/blog/2017/women-oss-lunch-linux-oss-summit.jpg" alt="Women in Open Source lunch OSSummit EU"/>
+<div class="image-credits">Women in Open Source lunch attendees (Image Credit: official OSSummit photographer)</div>
+
+During the summit, I attended both technical talks and talks focused on community and diversity in open source. I found both to be highly beneficial for my work as a developer and also as an organiser of [codebar](https://codebar.io/).
+
+Although I learned a ton throughout the conference, the information that had the most immediate impact on my life was finding out about the [Outreachy](https://www.outreachy.org/) programme. Outreachy was discussed during the Kernel Internship presentations and at [Vaishali Thakkar's](https://twitter.com/kernel_girl) talk on mentoring kernel newbies and then again at the Diversity Summit. Knowing that a programme exists is the essential first step to applying, so I am very grateful that I found out about Outreachy at the Open Source Summit. I messaged the organisers to see if I was eligible to apply (they have a clause that Google Summer of Code alumni are not eligible) and they assured me that RGSoC alumni were eligible for the Outreachy programme. What's more, they said I should apply in the next few days, as the window for applications was about to close. Examining the projects to apply to, I was very interested in [OpenTracing](http://opentracing.io/outreachy), specifically the task of splitting the Jaeger JavaScript OpenTracing library.
+
+Happily, I was already signed up to attend the optional Tracing Summit the next day. As you can imagine, I listened to the Tracing Summit talks with rapt attention. Then it was time to fly home and work very hard on the application. Fortunately, I was accepted onto the Outreachy programme and will be contributing to the Jaeger JavaScript OpenTracing library!
+
+Being at the Linux Foundation's Open Source Summit was an incredible experience, and it has led to opportunities that I had not previously imagined. I had a wonderful time, learned an enormous amount, made new friends, and got to spend more time with my super RGSoC supervisor, [Inês Coelho](https://twitter.com/ines_opcoelho).
+
+<img src="/img/blog/2017/ines-vaishali-kara-ossummit-prague.jpg" alt="Inês, Vaishali, and Kara"/>
```

Comment:
This image reference is broken because of the extension. Please fix that.

Reply:
Not sure what you mean -- this renders fine when I view it locally.